includes/acl/core/memory.h

```diff
@@ -72,15 +73,23 @@ namespace acl
 
 		virtual void* allocate(size_t size, size_t alignment = DEFAULT_ALIGNMENT)
 		{
+#ifndef _WIN32
+			return aligned_alloc(size, alignment);
```

Comment:
This bit is part of C11 standard. MSVC is very picky about which parts of C it decides to support

Reply:
Yeah, I really wanted to be able to use those but sadly msvc isn't complient :( yet...